code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -120,7 +105,7 @@
 		if(M.buckled)
 			continue
 
-		moveMob(M, src)
+		M.forceMove(src)

```

Comment:
make sure this doesn't bring back the infini-slip thing with soap/banana peels by opening and closing the closet repeatedly.